doorman/views.py

```diff
@@ -108,6 +109,10 @@ def add_query():
     form.set_choices()
 
     if form.validate_on_submit():
+        if not validate_osquery_query(form.sql.data):
+            flash(u'Invalid osquery query: "{}"'.format(form.sql.data), 'danger')
```

Comment:
Use numbered format specifiers. Not marrying myself to Python 2.6 support, but for someone who tries to run it will raise errors.